language/phpmailer.lang-id.php

```diff
@@ -5,22 +5,22 @@
  * @author Cecep Prawiro <cecep.prawiro@gmail.com>
  */
 
-$PHPMAILER_LANG['authenticate']         = 'Kesalahan SMTP: Tidak dapat mengautentikasi.';
+$PHPMAILER_LANG['authenticate']         = 'Kesalahan SMTP: Tidak dapat mengotentikasi.';
 $PHPMAILER_LANG['connect_host']         = 'Kesalahan SMTP: Tidak dapat terhubung ke host SMTP.';
 $PHPMAILER_LANG['data_not_accepted']    = 'Kesalahan SMTP: Data tidak diterima peladen.';
 $PHPMAILER_LANG['empty_message']        = 'Isi pesan kosong';
 $PHPMAILER_LANG['encoding']             = 'Pengkodean karakter tidak dikenali: ';
 $PHPMAILER_LANG['execute']              = 'Tidak dapat menjalankan proses : ';
 $PHPMAILER_LANG['file_access']          = 'Tidak dapat mengakses berkas : ';
 $PHPMAILER_LANG['file_open']            = 'Kesalahan File: Berkas tidak bisa dibuka : ';
-$PHPMAILER_LANG['from_failed']          = 'Alamat pengirim berikut mengakibatkan error : ';
-$PHPMAILER_LANG['instantiate']          = 'Tidak dapat menginisialisasi fungsi email';
-$PHPMAILER_LANG['invalid_address']      = 'Gagal terkirim, alamat email tidak valid : ';
+$PHPMAILER_LANG['from_failed']          = 'Alamat pengirim berikut mengakibatkan kesalahan : ';
+$PHPMAILER_LANG['instantiate']          = 'Tidak dapat menginisialisasi fungsi surel';
+$PHPMAILER_LANG['invalid_address']      = 'Gagal terkirim, alamat surel tidak benar : ';
 $PHPMAILER_LANG['provide_address']      = 'Harus disediakan minimal satu alamat tujuan';
-$PHPMAILER_LANG['mailer_not_supported'] = 'Mailer tidak didukung';
-$PHPMAILER_LANG['recipients_failed']    = 'Kesalahan SMTP: Alamat tujuan berikut menghasilkan error : ';
+$PHPMAILER_LANG['mailer_not_supported'] = 'Pengirim tidak didukung';
+$PHPMAILER_LANG['recipients_failed']    = 'Kesalahan SMTP: Alamat tujuan berikut menghasilkan kesalahan : ';
 $PHPMAILER_LANG['signing']              = 'Kesalahan dalam tanda tangan : ';
 $PHPMAILER_LANG['smtp_connect_failed']  = 'SMTP Connect() gagal.';
-$PHPMAILER_LANG['smtp_error']           = 'Kesalahan peladen SMTP : ';
+$PHPMAILER_LANG['smtp_error']           = 'Kesalahan pada pelayan SMTP : ';
 $PHPMAILER_LANG['variable_set']         = 'Tidak berhasil mengatur atau mengatur ulang variable : ';
 //$PHPMAILER_LANG['extension_missing']    = 'Extension missing: ';
```

Comment:
Could you add this missing translation too?

Reply:
you mean extension_missing ?

Reply:
Yes

Reply:
of course with pleasure